src/lib/queryParser.js

```diff
@@ -29,11 +29,11 @@ module.exports = function(queryString, locale) {
     presets = currency.presets[frequency];
   }
 
-  if (test && toString.call(test) === "[object Array]") {
+  if (test && test.join && test.map) {
```

Comment:
Is this effectively a browser compatibility check?

Reply:
the `test.map` is probably not even required. The code that it wraps is for "if this was an array, turn it into a string", so just testing for "does `test.join` exist?" is probably sufficient.

Reply:
removed test.map